jvm/src/main/kotlin/app/cash/trifle/internal/validators/X509CertChainValidator.kt

```diff
@@ -43,7 +44,21 @@ internal class X509CertChainValidator(certAnchor: Certificate, date: Date? = nul
       return false
     }
 
-    PATH_VALIDATOR.validate(X509FACTORY.generateCertPath(x509Certs), pkixParams)
+    try {
+      PATH_VALIDATOR.validate(X509FACTORY.generateCertPath(x509Certs), pkixParams)
```

Comment:
This doesn't return anything worth checking? Only throws?

Reply:
actually I take it back - it's not boolean this is a JCA method which returns the following [result type](https://docs.oracle.com/javase/8/docs/api/java/security/cert/PKIXCertPathValidatorResult.html) which our logic doesnt use since we are only validating:
All PKIXCertPathValidatorResult objects contain the valid policy tree and subject public key resulting from the validation algorithm, as well as a TrustAnchor describing the certification authority (CA) that served as a trust anchor for the certification path.